### PR TITLE
PROD321-3713/sell-us-your-ca

### PR DESCRIPTION
**Pull request by estradag87**
[PROD321-3713/sell-us-your-car](https://321ignitionproj.atlassian.net/browse/PROD321-3713)

- Chelsea Motors - 4.1 Sell Us Your Car - Drop in New Images
- Fix the logos
- Chelsea Motors - 4.1 Sell Us Your Car - Title & Metadata

Captura de Pantalla 2020-07-28 a la(s) 5 39 59 p  m

### CODE REVIEW

File: _site/css/common.css [hunk 1]
```diff
@@ -8,6 +8,11 @@
   border-bottom: 1.5px solid #cfcfcf !important;
 }
 
+.fi {
+  font-size: 3.5rem;
+  color: #505050;
+}
+
```
Comment (GabrielaDiazB) [inline _site/css/common.css:15]: If this style is specific for this page, you can create a css file for this page and import it to the js file of this page.

File: _site/css/common.css [hunk 2]
```diff
@@ -8,6 +8,11 @@
   border-bottom: 1.5px solid #cfcfcf !important;
 }
 
+.fi {
```
Comment (vela91) [inline _site/css/common.css:11]: please don't add the fi class, use 
.fi-to-do,.fi-sell-car-icon,.fi-cash,.fi-prices {
  color: #----;
  font-size: ----;
}

File: _site/css/common.css [hunk 3]
```diff
@@ -8,6 +8,11 @@
...
+  font-size: 3.5rem;
```
Comment (vela91) [inline _site/css/common.css:12]: is is posible use px for add font-size. Thanks

File: _site/css/common.css (hunk 1 above)
Comment (estradag87) [reply _site/css/common.css]: done!

File: _site/css/common.css (hunk 2 above)
Comment (estradag87) [reply _site/css/common.css]: done!

File: _site/css/common.css (hunk 3 above)
Comment (estradag87) [reply _site/css/common.css]: done!

**Pull request merged**